build.ps1

```diff
@@ -158,15 +158,14 @@ if(-Not $SkipToolPackageRestore.IsPresent) {
     if((!(Test-Path $PACKAGES_CONFIG_MD5)) -Or

```

Comment:
Please remove this file from review or just overwrite changes

Reply:
sorry, I commit 6 files, how many files do you look at?

Reply:
commit id 1b582a5e4c3a01f54717702d9f4c6eb3e2013cf5

Reply:
@JamesYing I see 6 files but the PR does not have any functionality just adds downstreamHealthcheck to configuration which does not make sense if the system doesnt use it for something!